### ISSUE TITLE
send_keys() does not work

### ISSUE THREAD

**Issue by oscheller1** [appium/python-client#167]
Using Appium 1.6.5beta with Appium-python-client 0.24

I'm getting the following error when trying to send_keys().
E.g. `element.send_keys("Max")`

```
[HTTP] --> POST /wd/hub/session/f579c1be-babe-47d9-bb07-b79166a6073d/element/4/value {"text":"Max","value":["M","a","x"],"sessionId":"f579c1be-babe-47d9-bb07-b79166a6073d","id":"4"}
[debug] [MJSONWP] Bad parameters: BadParametersError: Parameters were incorrect. We wanted {"required":["value"]} and you sent ["text","value","sessionId","id"]
[HTTP] <-- POST /wd/hub/session/f579c1be-babe-47d9-bb07-b79166a6073d/element/4/value 400 3 ms - 106 
```

Problem occurs on both Android and iOS devices. Could this be a rather recent change on the Appium server side that is causing this?

**Comment by mykola-mokhnach** [comment]
This should be fixed by https://github.com/appium/appium-base-driver/pull/120, which is not merged yet.
@jlipps FYI

**Comment by jlipps** [comment]
sorry, dropped the ball on that PR. made some more comments.

**Comment by gekitz** [comment]
I'm trying to figure out the current state of this problem, since this is not working on our side. This is fixed on appium-base-driver what's the next step to get it to work in the python client?

**Comment by oscheller1** [comment]
A workaround for Appium 1.6.5 was to `pip install selenium==3.0.0`.

This issue seems to be fixed running Appium 1.7.0+ with most recent selenium 3.6.0 python package.

**Comment by langme** [comment]
same issue with 3.7.0 to resolve it downgrade 3.0.0 #
